### PR TITLE
Add SVG and PNG buttons to volcano plot in enrichments

### PR DESCRIPTION
**Pull request by ersinciftci**
Fixes https://github.com/cBioPortal/cbioportal/issues/4381
No Data download button as it's the same data with the table

### CODE REVIEW

File: src/pages/resultsView/enrichments/MiniScatterChart.tsx [hunk 1]
```diff
@@ -72,8 +87,16 @@ export default class MiniScatterChart extends React.Component<IMiniScatterChartP
         return (
             <div className="posRelative">
                 <div className="borderedChart inlineBlock">
-                    <VictoryChart containerComponent={<VictorySelectionContainer responsive={false}
-                        onSelection={(points: any, bounds: any, props: any) => this.handleSelection(points, bounds, props)} 
+                    <div className="btn-group" style={{position:'absolute', zIndex:10, right: 26 }} role="group">
```
Comment (alisman) [inline src/pages/resultsView/enrichments/MiniScatterChart.tsx]: @adamabeshouse  has a component for this now I think.

Comment (ersinciftci) [reply src/pages/resultsView/enrichments/MiniScatterChart.tsx]: Updated

File: src/shared/components/DownloadControls.tsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ interface IDownloadControlsProps {
     additionalLeftButtons?:ButtonSpec[],
     additionalRightButtons?:ButtonSpec[]
     dontFade?:boolean;
+    dontDisable?:boolean;
```
Comment (adamabeshouse) [inline src/shared/components/DownloadControls.tsx]: why would we want to have the buttons not be disabled if the svg isnt available?

Comment (ersinciftci) [reply src/shared/components/DownloadControls.tsx]: I thought it had a dependency on having a `<svg ref={this.svgRef}` wrapper, but forgot that `<VictorySelectionContainer containerRef={this.svgRef}` also takes a function, so fixed.

**Pull request merged**